src/System.IO/tests/BinaryReader/BinaryReaderTests.cs

```diff
@@ -74,6 +75,8 @@ private void ValidateDisposedExceptions(BinaryReader binaryReader)
             Assert.Throws<ObjectDisposedException>(() => binaryReader.Read());
             Assert.Throws<ObjectDisposedException>(() => binaryReader.Read(byteBuffer, 0, 1));
             Assert.Throws<ObjectDisposedException>(() => binaryReader.Read(charBuffer, 0, 1));
+            Assert.Throws<ObjectDisposedException>(() => binaryReader.Read(new Span<byte>(byteBuffer)));
+            Assert.Throws<ObjectDisposedException>(() => binaryReader.Read(new Span<char>(charBuffer)));
```

Comment:
Tests for the new APIs will need to be separated out into their own file conditionally included only for netcoreapp.

Reply:
Done.